lib/env.ts

```diff
@@ -15,6 +15,7 @@ class EnvironmentImpl implements ENV {
   public readonly platform = platform;
 
   public debug = false;
+  public pack = true;
```

Comment:
>public pack = true; [](start = 2, length = 19)

[yulong] consider design optional flag with default value to false, and move it to webgl backend part.